packages/tatum-doge/src/model/request/PrivateKeyOrSignatureId.ts

```diff
@@ -0,0 +1,22 @@
+import {IsNotEmpty, IsOptional, IsUUID, Length, Min, Validate, ValidateIf} from 'class-validator'
+import {SignatureIdValidator} from '@tatumio/tatum-core'
+
+export class PrivateKeyOrSignatureId {
```

Comment:
This class also already exists in tatum-core so you can delete this file and import it from core lib.